runtime/compiler/control/CompilationThread.cpp

```diff
@@ -760,6 +760,7 @@ bool TR::CompilationInfo::shouldDowngradeCompReq(TR_MethodToBeCompiled *entry)
 bool
 TR::CompilationInfo::createCompilationInfo(J9JITConfig * jitConfig)
    {
+      J9JITConfig::compilationRuntime = *_compilationRuntime;
```

Comment:
_compilationRuntime is not defined at this point. You must do the assignment after the compilationInfo object has been allocated.